src/config.zig

```diff
@@ -379,11 +379,6 @@ comptime {
 
     // The LSM tree uses half-measures to balance compaction.
     assert(lsm_batch_multiple % 2 == 0);
-
```

Comment:
I think we can leave these in — I know they are checked elsewhere, but they sort of serve as documentation for the permitted (default) values.